### PR TITLE
US-264 : Take into account new design for challenge item buttons

### PR DESCRIPTION
**Pull request by jbuget**

### CODE REVIEW

**Comment by twickham** [comment]
I've deployed this PR to http://264-design-challenge-buttons.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://264-design-challenge-buttons.pix.beta.gouv.fr. Please check it out

File: live/tests/acceptance/37-previsualiser-un-test.js [hunk 1]
```diff
@@ -81,8 +81,8 @@ describe('Acceptance | 37 - Prévisualiser un test |', function () {
       });
 
       it("37.7. un bouton pour accéder à l'épreuve suivante", function () {
-        const $validateButton = findWithAssert('.validate-button');
-        expect($validateButton.text()).to.contains('Valider');
+        const $validateButton = findWithAssert('.challenge-item-actions__validate-action');
+        expect($validateButton.text()).to.contains('Je valide');
       });
```
Comment (bdavidxyz) [inline live/tests/acceptance/37-previsualiser-un-test.js:85]: https://github.com/sgmap/pix-live/issues/143

File: live/tests/acceptance/4-demarrer-une-epreuve-qcu-test.js [hunk 2]
```diff
@@ -49,11 +49,11 @@ describe('Acceptance | 4 - Démarrer une épreuve |', function() {
   });
 
   it('4.4. affiche le bouton "Valider" permettant de sauvegarder la réponse saisie et de passer à l\'épreuve suivante ', function() {
-    expect(findWithAssert('.validate-button').text()).to.contains('Valider');
+    expect(findWithAssert('.challenge-item-actions__validate-action').text()).to.contains('Je valide');
   });
```
Comment (bdavidxyz) [inline live/tests/acceptance/4-demarrer-une-epreuve-qcu-test.js:54]: https://github.com/sgmap/pix-live/issues/143

**Comment by twickham** [comment]
I've deployed this PR to http://264-design-challenge-buttons.pix.beta.gouv.fr. Please check it out

**Pull request merged**